core/src/main/scala/org/apache/spark/storage/StorageUtils.scala

```diff
@@ -46,7 +42,7 @@ class StorageStatus(
    * Internal representation of the blocks stored in this block manager.
    *
    * We store RDD blocks and non-RDD blocks separately to allow quick retrievals of RDD blocks.
-   * These collections should only be mutated through the add/update/removeBlock methods.
+   * These collections should only be mutated through the addBlock method.
```

Comment:
I think this is pretty out of date now. I don't see any calls to `addBlock` outside of this class.